runtime/gc_vlhgc/VLHGCAccessBarrier.cpp

```diff
@@ -271,6 +271,20 @@ MM_VLHGCAccessBarrier::copyArrayCritical(J9VMThread *vmThread, GC_ArrayObjectMod
 	}
 }
 
+IDATA
+MM_VLHGCAccessBarrier::indexableDataDisplacement(J9VMThread *vmThread,  J9IndexableObject *src,  J9IndexableObject *dst)
```

Comment:
looks like double space after the commas (here and a few other spots where the method is declared/defined)